### PR TITLE
Logging of using of APIs 

### PR DESCRIPTION
**Pull request by ea-rus**
Log using of Twitter and Openai API:
- in/out to openai
- in to twitter api

## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #(issue)

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

File: mindsdb/integrations/handlers/twitter_handler/twitter_handler.py [hunk 1]
```diff
@@ -312,6 +312,7 @@ def call_twitter_api(self, method_name:str = None, params:dict = None):
                 else:
                     params['max_results'] = left
 
+            log.logger.debug(f'>>>twitter in: {method_name}({params})')
```
Comment (paxcema) [inline mindsdb/integrations/handlers/twitter_handler/twitter_handler.py:315]: Same as the above, maybe there is a cleaner alternative to `>>>`, but this is not critical

**Review by paxcema — CHANGES_REQUESTED** [review]
Just one important change, the other two are just formatting-related.

File: mindsdb/integrations/handlers/openai_handler/openai_handler.py [hunk 2]
```diff
@@ -275,6 +276,11 @@ def _submit_completion(model_name, prompts, api_key, api_args, args, df):
             else:
                 return _submit_normal_completion(kwargs, prompts, api_args)
 
+        def _log_api_call(params, response):
+            params2 = params.copy()
+            params2.pop('api_key', None)
+            log.logger.debug(f'>>>openai call: {params2}:\n{response}')
```
[the remark this answers is not in the document]
Comment (StpMax) [reply mindsdb/integrations/handlers/openai_handler/openai_handler.py:282]: disagree, tabs will look worse, especially when we will use log collector

**Review by StpMax — APPROVED** [review]

File: mindsdb/integrations/handlers/openai_handler/openai_handler.py [hunk 3]
```diff
@@ -275,6 +276,11 @@ def _submit_completion(model_name, prompts, api_key, api_args, args, df):
             else:
                 return _submit_normal_completion(kwargs, prompts, api_args)
 
+        def _log_api_call(params, response):
+            params2 = params.copy()
+            params2.pop('api_key', None)
```
[the remark this answers is not in the document]
Comment (ea-rus) [reply mindsdb/integrations/handlers/openai_handler/openai_handler.py:281]: done

File: mindsdb/integrations/handlers/openai_handler/openai_handler.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ea-rus) [reply mindsdb/integrations/handlers/openai_handler/openai_handler.py:283]: with >>> it will be easy to look for records in logs

**Pull request merged**